presto-main/src/test/java/io/prestosql/cost/TestFilterStatsCalculator.java

```diff
@@ -294,6 +297,9 @@ public void testAndStats()
 
         assertExpression("'a' IN ('b', 'c') AND unknownRange = 3e0")
                 .outputRowsCount(0);
+
+        assertExpression("CAST(NULL AS boolean) AND CAST(NULL AS boolean)").equalTo(zeroStatistics);
+        assertExpression("CAST(NULL AS boolean) AND (x < 0e0 AND x > 1e0)").equalTo(zeroStatistics);
```

Comment:
Does `(x < 0e0 AND x > 1e0)` returns 0 stats?

Reply:
Yes, will add as a separate